src/number-input/number.component.ts

```diff
@@ -161,6 +163,8 @@ export class NumberComponent implements ControlValueAccessor {
 	 * Sets the invalid text.
 	 */
 	@Input() invalidText: string | TemplateRef<any>;
+
+	@Input() translations = this.i18n.get().NUMBER;
```

Comment:
To get the full benefit of the `i18n` service this should be a few local variables with `this.i18n.getOverridable("NUMBER.WHATEVER")` and then some getters/setters for the inputs :+1: